src/templates/singleBusinessPage.js

```diff
@@ -0,0 +1,325 @@
+              <LabeledSection label={'What we do'}>
+                <Text>{businessDescription}</Text>
+              </LabeledSection>
+            )}
+
+            {story && (
+              <LabeledSection label={'Who we are'}>
+                <Text>{story}</Text>
+              </LabeledSection>
+            )}
+          </div>
+          <Box
+            padding={theme.spacing.base}
+            border={`2px solid ${theme.colors['rbb-black-000']}`}
+            borderRadius={'8px'}
+            height="fit-content"
+            hidden={!serviceArea && paymentTypes.length === 0}
+          >
+            <Heading
+              as="h3"
+              textAlign="center"
+              textTransform="uppercase"
+              fontFamily={theme.fonts['heading-slab']}
+              fontSize={theme.fontSizes.lg}
+              fontWeight="900"
+              marginBottom={theme.spacing.xs}
+            >
+              Details
+            </Heading>
+
+            {serviceArea && (
+              <>
+                <Heading
+                  as="h4"
+                  textTransform="uppercase"
+                  fontFamily={theme.fonts['heading-slab']}
+                  fontSize={theme.fontSizes.base}
+                  fontWeight="900"
+                >
+                  Area of Service
+                </Heading>
+                <Text paddingLeft={theme.spacing.xs}>{serviceArea}</Text>
+              </>
+            )}
+
+            {paymentTypes.length && (
+              <>
+                <Heading
+                  as="h4"
+                  textTransform="uppercase"
+                  fontFamily={theme.fonts['heading-slab']}
+                  fontSize={theme.fontSizes.base}
+                  fontWeight="900"
+                  marginTop={theme.spacing.xs}
+                >
+                  Payment Types
+                </Heading>
+                <Text paddingLeft={theme.spacing.xs} textTransform="capitalize">
+                  {paymentTypes.map(
+                    (type, idx) =>
+                      `${type} ${paymentTypes.length > idx + 1 ? ' ,' : ''}`
+                  )}
+                </Text>
+              </>
+            )}
+          </Box>
+        </Grid>
+
+        <Link
+          href={website}
+          target="_blank"
+          rel="noopener noreferrer"
+          alignSelf="center"
+        >
+          <Button variant="primary" maxWidth="266px" leftIcon="external-link">
+            Share profile
+          </Button>
+        </Link>
+
+        <Text as="small" alignSelf="center" width="245px" fontWeight={500}>
+          <div>Notice something wrong with this listing?</div>
+          <Link
+            href="mailto:social@rebuildblackbusiness.com"
+            color={theme.colors.red[400]}
+          >
+            Send us an email and let us know.
+          </Link>
+        </Text>
+      </Stack>
+    </Layout>
+  );
+};
+
+export const query = graphql`
```

Comment:
this is more of a general comment than a suggestion, but doesn't it make more sense to turn these exported queries into hooks using `useStaticQuery` - similar to the linked doc below? 

https://www.gatsbyjs.com/docs/use-static-query/#composing-custom-usestaticquery-hooks

It can essentially help clean up some areas in our code and avoid the need to use the StaticQuery tag. This is obviously up for discussion so I'm happy to turn it into a GH issue instead.

Reply:
This is a good idea too - I wonder if I can combine this with the hook I've written on this page to pull from the api...

Reply:
yeah, we can have a general metadata hook instead of writing different metadata queries but this is probably case-by-case. I'm okay leaving this implementation for another PR, but up to you.